helpers/dmesg.py

```diff
@@ -48,16 +48,23 @@ def show(self):
         print(self.log)
 
     def _warn_count(self, warn_str):
-        match = re.findall(warn_str, self.log.decode())
-        return len(match)
+        return len(self._warn_match(warn_str))
 
-    def warn_count(self, warn_str):
+    def _warn_match(self, warn_str):
+        return re.findall(warn_str, self.log.decode())
+
+    def warn_count(self, warn_str: str) -> int:
```

Comment:
I got error here:
```
./run_tests.py -L
Traceback (most recent call last):
  File "./run_tests.py", line 10, in <module>
    from helpers import tf_cfg, remote, shell, control, prepare
  File "/tmp/tempesta-test/helpers/control.py", line 9, in <module>
    from . import tf_cfg, remote, error, nginx, tempesta, deproxy, stateful, dmesg
  File "/tmp/tempesta-test/helpers/dmesg.py", line 14, in <module>
    class DmesgFinder(object):
  File "/tmp/tempesta-test/helpers/dmesg.py", line 63, in DmesgFinder
    def warn_match(self, warn_str: str) -> list[str]:
TypeError: 'type' object is not subscriptable
root@ubuntu:/tmp/tempesta-test# ./run_tests.py -d
Traceback (most recent call last):
  File "./run_tests.py", line 10, in <module>
    from helpers import tf_cfg, remote, shell, control, prepare
  File "/tmp/tempesta-test/helpers/control.py", line 9, in <module>
    from . import tf_cfg, remote, error, nginx, tempesta, deproxy, stateful, dmesg
  File "/tmp/tempesta-test/helpers/dmesg.py", line 14, in <module>
    class DmesgFinder(object):
  File "/tmp/tempesta-test/helpers/dmesg.py", line 63, in DmesgFinder
    def warn_match(self, warn_str: str) -> list[str]:
TypeError: 'type' object is not subscriptable
```

Reply:
BTW this error indicates that your version of Python is 3+ years old ;)